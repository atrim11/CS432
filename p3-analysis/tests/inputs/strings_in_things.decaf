def int main() {
    int w;
    void op;
    bool cmp;
    op = 2 % 5 / "world";
    cmp = "hello" == "hello";
    cmp = "world" == "world";
    cmp = "NOT" != "equal";
    cmp = 1 != "str";
    cmp = "str" >= 1;
    cmp = 2 + 5 > "a";
    return "hello" + w;
}