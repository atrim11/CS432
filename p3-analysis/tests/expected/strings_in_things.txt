Void variable 'op' on line 3
Type mismatch: int expected but str found on line 5
Type mismatch: void is incompatible with int on line 5
Unsupported string equality check on line 6
Unsupported string equality check on line 7
Unsupported string equality check on line 8
Type mismatch: int is incompatible with str on line 9
Type mismatch: int expected but str found on line 10
Type mismatch: int expected but str found on line 11
Type mismatch: int expected but str found on line 12
